src/snapred/ui/workflow/NormalizationWorkflow.py

```diff
@@ -205,6 +206,7 @@ def _saveNormalizationCalibration(self, workflowPresenter):
 
         normalizationIndexEntry = NormalizationIndexEntry(
             runNumber=view.fieldRunNumber.get(),
+            useLiteMode=view.useLiteMode,
```

Comment:
I think this could use `self.useLiteMode` instead and avoid the use of signals, but I did not want to risk it.